features/LimitReuse.php

```diff
@@ -0,0 +1,152 @@
+<?php
```

Comment:
Whenever possible, try to ensure that file renames are performed as `git` moves, rather than deletes and re-adds.  This will minimize noise in change listings.